packages/functions/docs/modules/index.md

```diff
@@ -110,9 +140,11 @@ IP_HEADER_NAME
 
 `string` \| `undefined`
 
+The IP address of the request.
+
 #### Defined in
 
-[packages/functions/src/headers.ts:99](https://github.com/vercel/vercel/blob/main/packages/functions/src/headers.ts#L99)
+[packages/functions/src/headers.ts:111](https://github.com/vercel/vercel/blob/main/packages/functions/src/headers.ts#L111)
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** use canonical link